### PR TITLE
Fixed some bugs I had using it

### PR DESCRIPTION
**Pull request by Ztarbox**
tried to use your version of 'devcert'.
I liked it very much but had some problems with it:

1. Windows has no $HOME env variable, so I set the shared property to nodes `os.homedir()`
2. You tried to read the CACert 'protected' (so tried to decrypt it). But it was never written protected. So I reverted this code to its original state and return just the path of the original `rootCACertPath`
3. 'password-prompt' is an ES6 module which cannot be 'default imported' without the flag `esModuleInterop: true` in tsconfig.json.
Cause you set that to `false` I just switched the import to required-syntax.
Also described here: https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-function-d-ts.html

Hope you like it.

### CODE REVIEW

File: types/password-prompt.d.ts [hunk 1]
```diff
@@ -1 +1,11 @@
-export default function(prompt: string): Promise<string>;
```
Comment (mike-north) [inline types/password-prompt.d.ts:1]: to be honest I'm kind of surprised that this worked (and it seems to have). Can you say more about why this change was necessary? I note that `esModuleInterop` is explicitly false in this project, and `allowSyntheticDefaultImports` was not enabled.

It seems that for at least some verion(s) of `password-prompt`, consumption of a default export was fine. 

What changed?

**Review by mike-north — CHANGES_REQUESTED** [review]

**Comment by mike-north** [comment]
Closing in favor of #27 and #28, which address many of the points raised here

File: package.json [hunk 2]
```diff
@@ -50,21 +50,21 @@
     "@types/glob": "^5.0.34",
     "@types/lodash": "^4.14.92",
     "@types/mkdirp": "^0.5.2",
-    "@types/node": "^8.5.7",
+    "@types/node": "^13.9.2",
```
[the remark this answers is not in the document]
Comment (Ztarbox) [reply package.json:53]: Fine for me, didn't think about it this way.

File: package.json [hunk 3]
```diff
@@ -50,21 +50,21 @@
...
+    "@typescript-eslint/parser": "^2.24.0",
     "eslint": "^6.8.0",
     "eslint-config-prettier": "^6.10.0",
     "eslint-plugin-async-await": "^0.0.0",
     "eslint-plugin-node": "^11.0.0",
     "eslint-plugin-prettier": "^3.1.2",
     "prettier": "^1.19.1",
     "standard-version": "^4.3.0",
-    "typescript": "^3.7.5"
+    "typescript": "^3.8.3"
```
[the remark this answers is not in the document]
Comment (Ztarbox) [reply package.json:67]: No there wasn't, I think.

File: types/password-prompt.d.ts (hunk 1 above)
Comment (Ztarbox) [reply types/password-prompt.d.ts:1]: Almost a year old. Have no idea anymore
